test/e2e/common/pods.go

```diff
@@ -59,7 +59,7 @@ const (
 	maxBackOffTolerance  = time.Duration(1.3 * float64(kubelet.MaxContainerBackOff))
 	podRetryPeriod       = 1 * time.Second
 	podRetryTimeout      = 1 * time.Minute
-	podReadyTimeout      = 1 * time.Minute
```

Comment:
I only see this once, we do see the Pod was scheduled quite quickly, but never progressed to Running.
It may have been a slow pull, but it's not clear that we can see that via the logs.

> Nov 18 13:28:04.312: INFO: observed Pod pod-test in namespace pods-5897 in phase Pending conditions []
> Nov 18 13:28:04.360: INFO: observed Pod pod-test in namespace pods-5897 in phase Pending conditions
> [{PodScheduled True 0001-01-01 00:00:00 +0000 UTC 2020-11-18 13:28:04 +0000 UTC  }]
> Nov 18 13:29:04.265: FAIL: failed to see Pod pod-test in namespace pods-5897 running

Reply:
Checking the [graph metrics for this e2e test](https://testgrid.k8s.io/sig-release-master-blocking#gce-cos-master-default&width=5&include-filter-by-regex=should%20run%20through%20the%20lifecycle%20of%20Pods%20and%20PodStatus&graph-metrics=test-duration-minutes) the test run that flaked looks to be due to fluctuations in the cluster environment. Three other tests also failed in the same run.

The current timeout is a lot more optimistic  than the values set in [wait.go](https://github.com/kubernetes/kubernetes/blob/379ed6644da86d2e87e33676a6145fc91805e4bf/test/e2e/framework/pod/wait.go#L54) and waiting a bit more will help the test deal with any fluctuations in the clusters state.

Reply:
interestingly, the place where this test is failing has been due to watch timeouts failing:

https://github.com/kubernetes/kubernetes/blob/5e44d8ebafc866e132530fa1328872561ba5f6da/test/e2e/node/pods.go#L307-L317

And that has a timeout of 5 minutes

Reply:
Sorry, realizing that this PR and https://github.com/kubernetes/kubernetes/issues/96565 are actually referring to two different tests, but the root cause seems likely similar.